src/include/sof/audio/ipc-config.h

```diff
@@ -61,22 +101,26 @@ struct ipc_config_src {
 /* generic ASRC component */
 struct ipc_config_asrc {
 	/* either source or sink rate must be non zero */
-	uint32_t source_rate;           /**< Define fixed source rate or */
-					/**< use 0 to indicate need to get */
-					/**< the rate from stream */
-	uint32_t sink_rate;             /**< Define fixed sink rate or */
-					/**< use 0 to indicate need to get */
-					/**< the rate from stream */
-	uint32_t asynchronous_mode;     /**< synchronous 0, asynchronous 1 */
-					/**< When 1 the ASRC tracks and */
-					/**< compensates for drift. */
-	uint32_t operation_mode;        /**< push 0, pull 1, In push mode the */
-					/**< ASRC consumes a defined number */
-					/**< of frames at input, with varying */
-					/**< number of frames at output. */
-					/**< In pull mode the ASRC outputs */
-					/**< a defined number of frames while */
-					/**< number of input frames varies. */
+	uint32_t source_rate;	/**< Define fixed source rate or
+				  *  use 0 to indicate need to get
+				  *  the rate from stream
+				  */
+	uint32_t sink_rate;	/**< Define fixed sink rate or
+				  *  use 0 to indicate need to get
+				  *  the rate from stream
+				  */
+	uint32_t asynchronous_mode;     /**< synchronous 0, asynchronous 1
+					  *  When 1 the ASRC tracks and
+					  *  compensates for drift.
+					  */
+	uint32_t operation_mode; /**< push 0, pull 1, In push mode the
+				   *  ASRC consumes a defined number
+				   *  of frames at input, with varying
+				   *  number of frames at output.
+				   *  In pull mode the ASRC outputs
+				   *  a defined number of frames while
+				   *  number of input frames varies.
+				   */
```

Comment:
remove this change, this is just an indentation change unrelated to this PR, or push it to a separate patch.